src/frontend/src/app/scheduled-process/scheduled-process.component.ts

```diff
@@ -97,19 +77,17 @@ export class ScheduledProcessComponent implements OnInit {
       error => { console.log('error', error); });
   }
 
-  public getSchedulesProcesses(): void {
-    this.scheduleProcess = [];
-    this.schedulesService.getScheduledProcess().
-      subscribe(
-      data => {
-         if (data.error) {
-          this.alertService.error(data.error.message);
-          return;
-        }
-        this.scheduleProcess = data.processes;
-        console.log('This is the getScheduleProcess ', this.scheduleProcess);
-      },
-      error => { console.log('error', error); });
+  /**
+   * Get ScheduleType, ScheduleProcess from create-schedule.component.ts
+   * @param data:  record {ScheduleType, ScheduleProcess} transmitted from
+   * child compoent create-schedule.component.ts
+   */
+  public setScheduleProcessAndType(data) {
+    if (_.some(data, { name: 'STARTUP' })) {
```

Comment:
explain the logic?

Reply:
_.some(collection, [predicate=_.identity]), Returns true if any element passes the predicate check, else false. 
data -> Array of Types or Process 
predicate -> Value of object holds in _data_ array to verify if it is a type array or a process array

Reply:
Yupe and this is why question raised, why are you hard coding the check to find what data array contains. why don't we make `setScheduleProcess` and `setScheduleType` methods